rclcpp/resource/logging.hpp.em

```diff
@@ -91,7 +91,7 @@ def is_supported_feature_combination(feature_combination):
  */
 #define RCLCPP_@(severity)@(suffix)(logger, @(''.join([p + ', ' for p in get_macro_parameters(feature_combination).keys()]))...) \
   static_assert( \
-    ::std::is_same<typename std::remove_reference<decltype(logger)>::type, \
+    ::std::is_same<typename std::remove_reference<std::remove_const<decltype(logger)>::type>::type, \
```

Comment:
too bad `std::remove_cvref` is not in there yet, but I think `std::remove_cv` is a better replacement for it which also covers volatile.

Reply:
+1 for `std::remove_cv`

Reply:
284cd94c2c30eca5d0f19d1c7d9a7d14ec24d23b there you go